db/migrate/20230718141415_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] According to the ERD:

![image](https://github.com/Elhambasir/blog-app/assets/77069011/d3699c2d-5326-4651-9afa-ce52d8aad6eb)

A Post should have `CommentsCounter` and `LikesCounter`. Also, it should have a reference to the `Users` table under the name of `AuthorId`. Please consider applying this change! 🤓 🔧